src/connections/MicrosoftXmlHttpRequestConnection.js

```diff
@@ -9,6 +9,7 @@ define(["structures/Response", "structures/CAPIError"], function (Response, CAPI
      *
      * @class MicrosoftXmlHttpRequestConnection
      * @constructor
+     * @deprecated
```

Comment:
maybe `@deprecated since 1.2`. I would also add in a warning when this object instantiated (similar to https://github.com/ezsystems/ez-js-rest-client/pull/71/files#diff-1cddad75741914589caf23a4815ed7abR1421)